.idea/codeStyles/Project.xml

```diff
@@ -5,6 +5,22 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="999" />
     </JavaCodeStyleSettings>
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
```

Comment:
We should not change this file. 
This change happens sometimes when our Android Studio coding style is not as same as the project. You can undo these changes and check `Editor->Code Style->scheme->Project`. Also, you can set code style for Java, Kotlin, XML to project. 

This way Android Studio will pick the code style from the project files and do not enforce the Android Studio code style on the project.

Reply:
Ok I'll do that

Reply:
> @anandwana001 I made the changes, can you please check and merge if it's ok?

The codestyle thing has not been updated yet. Please check the file and I will review again.

Reply:
I've updated the codestyle as you mentioned. 
![image](https://user-images.githubusercontent.com/54766268/94466307-e027bc00-01de-11eb-8cf6-f34920196ad4.png)

Reply:
looks good, Thanks